package.json

```diff
@@ -113,16 +113,17 @@
     "style-loader": "^0.13.2",
     "styled-components": "^4.0.3",
     "typeface-lato": "^0.0.75",
-    "webpack": "^4.30.0",
-    "webpack-bundle-analyzer": "^3.3.2",
-    "webpack-dev-middleware": "^3.1.3",
-    "webpack-hot-middleware": "^2.24.3",
+    "webpack": "^5.74.0",
+    "webpack-bundle-analyzer": "^4.6.1",
+    "webpack-dev-middleware": "^5.3.3",
```

Comment:
I'm getting the following error when in development mode - do you as well?
```
$ auspice develop
/Users/enigma/projects/nextstrain/auspice/node_modules/webpack-dev-middleware/node_modules/schema-utils/dist/validate.js:115
    throw new _ValidationError.default(errors, schema, configuration);
    ^

ValidationError: Invalid options object. Dev Middleware has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'logLevel'. These properties are valid:
 ...
 ```

Reply:
Yes I see this too. Looks like something with the Webpack update. Thanks for checking, maybe it'd be good to check that `auspice develop` runs properly in CI as well.

Reply:
The arises from behavior differences starting from [webpack-dev-middleware version 4.0.0](https://github.com/webpack/webpack-dev-middleware/blob/master/CHANGELOG.md?plain=1#L190).

I replaced ffe9375762677e2fa3ab53e51288be8d25783d6d with a different, more conservative upgrading approach which should fix this: 922581e022c6439be8186b26e99902f4bb935e01

Reply:
I'm still not able to run `auspice develop` -- the terminal now looks ok but the client won't load. Console error: `Uncaught (in promise) ReferenceError: module is not defined`.

Furthermore the console error is coming from `auspice.chunk.core-vendors.bundle.js line 7145 > eval:8:3` which indicates that the development mode is actually running from the (minimised) bundles rather than using source maps to link errors back to the actual src code.

Reply:
Ok I also see this with both NodeJS 14 and 18. Thanks for flagging!

Reply:
@jameshadfield that console error should be fixed now, see https://github.com/nextstrain/auspice/pull/1520#discussion_r987382659. And _I think_ the development mode is using the actual source code now.

Reply:
Nice one! Both the source maps and the hot reloading are now working as expected 😄